src/main/java/org/opentripplanner/api/resource/GraphPathToTripPlanConverter.java

```diff
@@ -288,13 +292,45 @@ private static void addAccessibilityScore(List<Leg> legs, State[][] states) {
                         computeAccessibilityScore(trip, currentLeg, nextLeg, request);
             }
             else if (currentLeg.mode.equals("WALK") && request.wheelchairAccessible) {
-                // at the moment we don't compute the walking score at all and add a dummy value instead
-                // in the future we will take the slope into account
-                currentLeg.accessibilityScore = 0.5f;
+                currentLeg.accessibilityScore = computeWalkingAccessibilityScore(request, legStates);
             }
         }
     }
 
+    private static float computeWalkingAccessibilityScore(RoutingRequest request, State[] legStates) {
```

Comment:
Needs JavaDoc, mentioning the lower and upper-bounds of the returned value.

Reply:
Added.